.github/ISSUE_TEMPLATE/2bug_report.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Bug Report
+about: Have you found a bug?
```

Comment:
```suggestion
about: Do you believe you have found a bug?
```

I'm not strongly for either wording, just wanted to throw this change out to see any thoughts.

Reply:
👍 on this change.  If I were a user and was unsure, I would feel more comfortable choosing this than the more definitive version.